### DISCUSSION TITLE
如何改善辨識結果

### DISCUSSION THREAD

**Discussion by aexss12378** [PaddlePaddle/PaddleOCR#13470] [Q&A] [answered]
![result](https://github.com/user-attachments/assets/5c8c529f-96f2-4daf-b43e-b07d4c6d0e82)
如圖，有一些字辨識錯誤
最好的方法是自己微調模型嗎
使用的偵測模型為：ch_PP-OCRv4_det_server_infer
           辨識模型為：chinese_cht_PP-OCRv3_rec_infer

**Comment by GreatV** [comment, marked as answer]
why not use ch_PP-OCRv4_rec_server_infer model?

**Reply by aexss12378** [reply]
Thanks.I thought i needed to use chinese_cht rec model to recongize Traditional Chinese.
